hnclient-v2/HNClient-Service.xml

```diff
@@ -0,0 +1,26 @@
+
+
+<!--
+ This is an example of a HNCLient Windows Service Wrapper configuration, which includes only mandatory options.
+ 
+ This configuration file should be placed near the WinSW executable, the name should be the same.
+ E.g. for HNClient-Service.exe the configuration file name should be  HNClient-Service.xml
+ 
+ You can find more information about the configuration options here: https://github.com/winsw/winsw/releases/tag/v2.11.0 sample-minimal.xml
+ and sample-allOptions.xml
+ 
+-->
+<configuration>
+  
+  <!-- ID of the service. It should be unique across the Windows system-->
+  <id>hnclient-service-new</id>
+  <!-- Display name of the service -->
+  <name>HNClient App Service</name>
+  <!-- Service description -->
+  <description>HNClient Application running as Window Service</description>
+  
+  <!-- Path to the executable, which should be started -->
+  <executable>java</executable>
+  <arguments>-jar %BASE%\HNClient-Service-jar-with-dependencies.jar</arguments>
```

Comment:
This relates to my comment about choice of frameworks but I wonder if we should just use a simple name for the executable jar. E.g. hns-client.jar. I'm not sure if the HNClient-Service part here is directly output from maven (which I don't believe to be the case) or it's something specific to the wrapper.